playbooks/templates/ucsm_job_local_conf.j2

```diff
@@ -14,3 +15,10 @@ ucsm_username=admin
 ucsm_password=Cisc0123
 ucsm_host_list={{ ansible_hostname }}:devstack-{{ ansible_hostname[-6:] }}
 ucsm_https_verify=False
+
+[[post-config|$TEMPEST_CONFIG]]
+[ucsm]
+service_profile=devstack-{{ ansible_hostname[-6:] }}
+ucsm_ip=192.133.149.20
```

Comment:
change looks fine.  An enhancement would be to put the UCSM IP and creds in group_vars/all.yaml to make the template usable for multiple UCSMs (future-proof-ish).  doesn't really matter at the moment, though

Reply:
good idea, I had to update this patch anyway (post-config should have been test-config) so I've added it :)